src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -405,6 +405,19 @@ export function getUiSettingDefaults() {
       }),
       category: ['search'],
     },
+    'courier:batchSearches': {
+      name: i18n.translate('kbn.advancedSettings.courier.batchSearchesTitle', {
+        defaultMessage: 'Batch concurrent searches in a single requests',
```

Comment:
I think you could shorten this even further without losing the meaning:
```suggestion
        defaultMessage: 'Batch concurrent searches',
```
That way it only takes up 1 line in the UI.